### PR TITLE
new update for UCSC Transient Team web pages at footer

### PR DESCRIPTION
**Pull request by SamLaiiii**
Fixes Issue #32  .

## Checklist

Please check all that apply to your proposed changes

- [x] HTML code change
- [ ] Added package dependency
- [ ] Added data
- [ ] Changed django model
- [ ] Documentation change
- [ ] Added or changed TaskRunner

### CODE REVIEW

**Comment by SamLaiiii** [comment]
I think I push the wrong file, let me do it with the right one

File: app/host/templates/host/base.html [hunk 1]
```diff
@@ -64,11 +64,12 @@
 
         <footer class="footer py-3 bg-light">
             <div class="container">
-                <span class="text-muted">Report bugs and issues on Github
+                <span class="text">Report bugs and issues on Github
                     <a href="https://github.com/astrophpeter/blast/issues">here</a>
 
-                </span>
-             </div>
+                <span style="margin-left:2px"><span class="text">| UCSC Transitent team web pages
```
Comment (astrophpeter) [inline app/host/templates/host/base.html]: Looks good! Can you please change the class here to text-muted from text

File: app/host/templates/host/base.html [hunk 2]
```diff
@@ -64,11 +64,12 @@
...
+                    <a href="https://transients.ucsc.edu/">here</a>
+
         </footer>
```
Comment (astrophpeter) [inline app/host/templates/host/base.html:75]: Can you please put </div> on this line, you deleted it :)

**Review by astrophpeter — CHANGES_REQUESTED** [review]
Looks good overall, just some minor changes please :)

**Review by astrophpeter — APPROVED** [review]
Looks great!

**Pull request merged**